### DISCUSSION TITLE
Welcome to SkyPilot Discussions!

### DISCUSSION THREAD

**Discussion by romilbhardwaj** [skypilot-org/skypilot#1015] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community. We are excited to have you here!

**Comment by maulikr** [comment]
I am testing skypilot for my organization and working on a PoC which involves SkyPilot.
Is there a way only to check quotes on different clouds than directly deploying a cluster? My PoC has limited budget and deploying a cluster could prove to be a little costly.

**Reply by concretevitamin** [reply]
Hi @maulikr: if you'd like to check prices of different clouds without launching:
- `sky launch`: will show these prices, with a confirmation prompt asking if you'd like to launch
- `sky show-gpus`: will show prices for GPU/TPU instances

We don't have a programmatic API to query prices of all resources in a task's search space yet. Could be interesting to have.

**Comment by wpsystems** [comment]
Hi, I notice that the python launch api has a OptimizeTarget.TIME option.  Can you provide more info about this optimization option?  Thanks,

**Reply by WoosukKwon** [reply]
Hi @wpsystems, if you use set TIME as the target, the SkyPilot optimizer selects the types of cloud VMs in a way that the end-to-end latency of your DAG (i.e., tasks) is minimized. You can use this option in our Python API; register the time estimators for your tasks by set_time_estimator and change the optimize_target argument of sky.launch.
